app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,7 @@
 
 <% content_for :content do %>
 
-  <%= breadcrumbs style: :bootstrap %>
+  <%= breadcrumbs style: :bootstrap, display_single_fragment: true %>
```

Comment:
I'm not sure about this - it seems weird to me - you're already on the page and there's nothing to go back to, so why not only display breadcrumbs when there's a trail? But it's in the prototype, so let's go with it.